compiler/src/dmd/func.d

```diff
@@ -217,6 +217,8 @@ private struct FUNCFLAG
     bool hasAlwaysInlines;   /// Contains references to functions that must be inlined
     bool isCrtCtor;          /// Has attribute pragma(crt_constructor)
     bool isCrtDtor;          /// Has attribute pragma(crt_destructor)
+    bool hasEscapingSiblings;/// Has sibling functions that escape
+    bool computedEscapingSiblings; /// `hasEscapingSiblings` has been computed
```

Comment:
I wonder if using https://github.com/dlang/dmd/blob/master/compiler/src/dmd/root/optional.d to keep it all in one variable would be a bit cleaner.

Reply:
That can't be put in bit fields right now

Reply:
:facepalm: I completely missed that this was in FUNCFLAG and not FuncDeclaration; still learning my way around the codebase.